README.md

```diff
@@ -741,43 +741,18 @@ Once you customize the above template/examples for your environment, create/upda
 
 ## Run the software deployment
 
-* Wait for the OpenStackConfigGenerator to finish the playbook rendering job
+* Wait for the OpenStackConfigGenerator to finish the playbook rendering job.
 
-* In the openstackclient pod
+* Obtain the hash/digest of the latest OpenStackConfigVersion.
 
-a) check for new version of rendered playbooks and accept them
+* Install the pre-requisites on overcloud systems for ceph-ansible
```

Comment:
we can remove this step as it is no longer required. lvm2 is now part of the ooo bootstrap package list.

Reply:
done